### ISSUE TITLE
[Question] Opening Spotify tracklink via Spotube

### ISSUE THREAD

**Issue by Yasuke976** [aleksey-saenko/MusicRecognizer#45]
As that's the only online music service that I use, would it be possible to add [Spotube](https://github.com/KRTirtho/spotube) as a musical service after recognition of a song?
BTW thanks for this great app!

**Comment by aleksey-saenko** [comment]
Hello. Spotube is not a music service, but a Spotify client. I believe it should be able to handle original Spotify links. Please go to Spotube App info -> Open by default, and make sure that `open.spotify.com` is checked.

<img src='https://github.com/aleksey-saenko/MusicRecognizer/assets/109096599/deb92910-df21-495f-9fc6-f76f747b0d8a' width='200'>

**Comment by Yasuke976** [comment]
Sorry I didn't mentionned it, Spotube is parametered as above. When a song is recognized, I select Spotify to open it in Spotube, Spotube opens, but stays on the home screen, without leading to the recognized song.

**Comment by aleksey-saenko** [comment]
I have found the [issue](https://github.com/KRTirtho/spotube/issues/508#issuecomment-1877780978) in the Spotube repository, and from my understanding, this particular feature has not been implemented in the Spotube app. Are you able to open the track page using a track link? For instance, you can try opening [this link](https://open.spotify.com/track/7L3qcsnGgEDjIbZutyWgR8) in a browser or messenger and check redirecting to track page.

**Comment by Yasuke976** [comment]
While reading the issue you mentioned on Spotube I saw at the end that the feature was implemented and coming in the future version.
I then installed the latest Spotube nightly and everything worked as expected, the recognized song opened directly in Spotube.
So everyrhing is fine for now, I just have to wait for the next official Spotube version to check if it's all good.
Thanks for the quick and efficient help!
